pkg/controller/statefulset/stateful_set.go

```diff
@@ -48,13 +48,13 @@ import (
 // controllerKind contains the schema.GroupVersionKind for this controller type.
 var controllerKind = apps.SchemeGroupVersion.WithKind("StatefulSet")
 
-// StatefulSetController controls statefulsets.
-type StatefulSetController struct {
+// Controller controls statefulsets.
+type Controller struct {
```

Comment:
Please don't change those, it's not desirable to blindly follow golint failures.

Reply:
Thank you so much for taking a look, I was myself in a similar impression. And asked the same question (here)[https://github.com/kubernetes/kubernetes/issues/68026#issuecomment-675431654].

 Is there is a way to ignore only a specific failure of Go lint, I am aware that we can have a comment to ignore an entire file. 

Shall I just close this PR? or is there a reason to hold - just trying to be sure.

Reply:
If this is the only change in your PR, then I guess so.